lib/scrub_fields.rb

```diff
@@ -94,24 +79,32 @@ def ocn(str)
     str.strip!
     candidates = str.split(OCN_SPLIT_DELIM)
 
+    # TODO: maybe we don't care about this?
+    # I've found it to be a good indicator that the record wasn't
+    # exported properly, but there are legit cases of this I guess.
     if candidates.size > MAX_NUM_ITEMS
       count_x(:too_many_ocns)
-      @logger.error "Too many items (#{candidates.size}) in ocn #{str}"
+      Services.scrub_logger.warn "Too many ocns (#{candidates.size}) in ocn #{str}"
+    end
+
+    if candidates.size > 1
+      count_x(:multi_ocn_record)
     end
 
-    # DO WHILE MAYBE COMEFROM reject_value
+    # PLEASE DO WHILE MAYBE COMEFROM reject_value
     candidates.each do |candidate|
       catch(:rejected_value) do
```

Comment:
Maybe worth extracting the contents of the block to a separate method?

Reply:
Done.